src/index.ts

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import http from 'http';
+import bodyParser from 'body-parser';
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+import cors from 'cors';
+import mongoose, { modelNames } from 'mongoose';
+import router from './router';
+
+const app = express();
+
+app.use(cors({
+    credentials: true,
+}))
+
+app.use(compression());
+app.use(cookieParser());
+app.use(bodyParser.json());
+
+const server =http.createServer(app);
+
+server.listen(8080, () => {
+    console.log('Serveris ir labs klausītājs')
+});
+
+const MONGO_URL = 'mongodb+srv://Ozokino:BLcRHxFAZylQHqKZ@taskdata.mzba1uw.mongodb.net/?retryWrites=true&w=majority&appName=TaskData';
```

Comment:
remove this URL, it is sensitive data, it should be stored in either environment variables or an env file